BLOG.md

```diff
@@ -0,0 +1,49 @@
+# Revisiting Force-Horse: Making a Legacy Plugin Agnostic and Usable
```

Comment:
Thanks!
It is legacy, just because AngularJS is legacy, see:
> Legacy code refers to an application system source code type that is no longer supported

Yeah, you kind of mixed some ES6 in there, just a bit.